app/controllers/projects/samples/file_imports_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Projects
+  module Samples
+    # Controller actions for Project Samples File Import Controller
+    class FileImportsController < Projects::ApplicationController
+      include SampleFileImportActions
+
+      before_action :ensure_enabled
+
+      respond_to :turbo_stream
+
+      private
+
+      def namespace
+        @namespace = @project.namespace
+      end
+
+      def ensure_enabled
+        render :not_found unless Flipper.enabled?(:batch_sample_file_import)
```

Comment:
This can be changed to `not_found unless Flipper.enabled?(:batch_sample_file_import)`. We have a `not_found` method in the application_controller. This applies to the groups samples file_imports_controller as well

Reply:
updated in https://github.com/phac-nml/irida-next/pull/962/commits/adb26aca5a00bd55ff9b02367c6091c155054776